src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -5317,7 +5317,7 @@ def test_dynkernelarguments_acc_args_4():
 
     '''
     _, info = parse(os.path.join(BASE_PATH,
-                                 "19.25_single_stencil_cross2d.f90"))
+                                 "19.26_single_stencil_cross2d.f90"))
```

Comment:
The number of the file was duplicated with another at some point since it went on so i changed mine to avoid duplicate numbering. The filename change can be seen at the bottom of the files changed list.